crm/modules/member/install.inc.php

```diff
@@ -161,4 +161,19 @@ function member_install($old_revision = 0) {
             }
         }
     }
+    if ($old_revision < 7) {
+        // Alter member table
+        $sql = '
+            ALTER TABLE `member`
+              ADD COLUMN `address1` varchar(255) NOT NULL
+              , ADD COLUMN `address2` varchar(255) NOT NULL
+              , ADD COLUMN `address3` varchar(255) NOT NULL
+              , ADD COLUMN `town_city` varchar(255) NOT NULL
+              , ADD COLUMN `zipcode` varchar(255) NOT NULL
```

Comment:
does zipcode really need 255 chars?  Do we need to care about how much space this takes up in the DB?  Or does varchar correctly only allocate UP TO 255 chars based on the length of all values in the column for that table?

Reply:
Also, I just copied what had been used when creating other tables/columns :P